tests/integration_tests/with_db/deployment/test_deploy_cli.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from click.testing import CliRunner
+import pyexasol
+import exasol.bucketfs as bfs
+from exasol.python_extension_common.cli.std_options import StdParams, get_cli_arg
+from exasol.python_extension_common.deployment.language_container_validator import temp_schema
+
+from exasol_transformers_extension.deploy import (
+    deploy_command, DEPLOY_SLC_ARG, BUCKETFS_CONN_NAME_ARG, get_bool_opt_name)
+from exasol_transformers_extension.deployment.language_container import export_slc
+from tests.integration_tests.with_db.test_upload_model import run_model_upload_test
+
+PATH_IN_BUCKET = 'te_end2end'
+BUCKETFS_CONN_NAME = 'TE_E2E_BFS_CONN'
+LANGUAGE_ALIAS = 'TE_E2E_LANG_ALIAS'
+
+
+def test_deploy_cli(pyexasol_connection,
+                    backend_aware_database_params,
+                    backend_aware_bucketfs_params,
+                    bucketfs_cli_args,
+                    cli_args):
+    """
+    This is an end-to-end test that should be moved into a separate group of tests.
```

Comment:
i dont really like this comment here. can you remove it and create a ticket instead?

Reply:
OK, will do.